src/components/TopBar/SaveBoard.tsx

```diff
@@ -43,12 +40,10 @@ const SaveBoard: React.FC = () => {
   const designsData = useStoreSelector((state) => state.courtSpecData.designsData);
   const designNames = useStoreSelector((state) => state.designName.nameList);
   const cancelRef = useRef(null);
-  const [nameCheck, setNameCheck] = useState<string>("exsited");
+  const [nameCheck, setNameCheck] = useState<string>("");
```

Comment:
If it has an initial value as a string, you don't need to define the type of string anymore, unless you are unsure(maybe need another type other than string later)